main.tf

```diff
@@ -1,7 +1,13 @@
+locals {
+  # Remove the var.domain_name from the subject_alternative_names as it doesn't need to be there.
```

Comment:
@Jamie-BitFlight 
I'm not clear on what _harm_ actually results if callers pass both `var.domain` and include that value somewhere in the list of `var.subject_alternative_names`.

Is there any harm?

Reply:
This seems to fall apart if you request a sub-sub san like 
domain: prod.mydomain.com
san *.uk.prod.mydomain.com